### ISSUE TITLE
About the three stage strategy

### ISSUE THREAD

**Issue by killwy** [gallenszl/CFNet#30]
In your paper,you mentioned that "we switch the activation function to Mish and prolong the pre-training process in the SceneFlow dataset for another 15 epochs".So,should the learning rate change in another 15 epochs?

**Comment by gallenszl** [comment]
Hello, for the prolonging 15 epochs, the learning rate is start at 0.001 and downscaled by 2 at epoch 10.

**Comment by killwy** [comment]
ok,thanks for your instant reply
